packages/sql-graphql/lib/entity-to-type.js

```diff
@@ -15,6 +15,11 @@ const ascDesc = new graphql.GraphQLEnumType({
   }
 })
 
+const limitType = new graphql.GraphQLScalarType({
+  name: 'LimitInt',
+  description: 'Default: 10. Max accept value: 100. If an higher value is provided will be capped to the actual accepted max value.'
+})
```

Comment:
Why was this change needed?

Reply:
In order to document the behavior behind the limit param in graphql's schema (there is a specular change for the swagger file).

Am I misunderstanding what we were discussing [here](https://github.com/platformatic/platformatic/pull/220#issuecomment-1288632086]?